slidetoact/src/main/java/com/ncorti/slidetoact/SlideToActView.kt

```diff
@@ -736,6 +749,29 @@ class SlideToActView @JvmOverloads constructor (
         }
     }
 
+    /**
+     * Private method to handle vibration logic, called when the cursor it moved to the end of it's path
+     * @throws SecurityException if bumpVibration is greater than 0 but the permission
+     * */
+    @SuppressLint("MissingPermission")
+    private fun handleVibration() {
```

Comment:
Looks like you don't call this method?

Reply:
Woops 😬
I think I didn't pushed the latest changes. I'm such a mess